verl/utils/dataset/rl_dataset.py

```diff
@@ -115,6 +115,12 @@ def __init__(
         self.need_tools_kwargs = config.get("need_tools_kwargs", False)
         self.filter_prompts = config.get("filter_prompts", True)
         self.serialize_dataset = False
+        self.processor_type = self.processor.image_processor.__class__.__name__ if self.processor is not None else None
```

Comment:
Is there any way to make this generalize?

Reply:
Sorry, I mean line 119, where says if self.processor_type == "MiniCPMVImageProcessor":

Reply:
It seems it's model specific